addon/templates/components/uni-multi-selector.hbs

```diff
@@ -0,0 +1,12 @@
+<div class="input-container">
+  <div class="uni-multi-selector">
+    {{#each options as |option|}}
+      {{uni-selector-button
+        name=name
+        groupValue=groupValue
+        value=option.value
+        label=option.label
```

Comment:
I would suggest changing from ```label``` to ```key``` (on ```option.label```) to be more explicit and have a more generic structure